src/components/chat/Input.tsx

```diff
@@ -0,0 +1,62 @@
+import { IconNames } from '@blueprintjs/icons';
+import * as React from 'react';
+
+import { controlButton } from '../commons';
+
+type StateProps = {
+  message: string;
+};
+
+interface InputProps {
+  onSubmit: (text: string) => void;
+}
+
+class Input extends React.Component<InputProps, StateProps> {
+  public constructor(props: InputProps) {
+    super(props);
+    this.state = {
+      message: ''
+    };
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  public render() {
+    const sendButtonOpts = {
+      className: 'chat-send-button',
+      fullWidth: true
+    };
+
+    return (
+      <div className="Input">
+        <div className="input-zone">
+          <textarea
```

Comment:
Possible to use blueprintjs Textarea? https://blueprintjs.com/docs/#core/components/text-inputs.text-area

Reply:
Yea, I've made the switch.